sys-process/cronutils/cronutils-1.10.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
```

Comment:
Unused

Reply:
actually it's used in: emake CC="$(**tc-getCC**)" CFLAGS="-D_XOPEN_SOURCE=500 ${CFLAGS}"